index.js

```diff
@@ -57,14 +57,14 @@ module.exports = function (input) {
   if (!validAmount(amount) || !parsableUnit(unit)) {
     throw 'Can\'t interpret ' + (input || 'a blank string');
   }
-  if (unit === '') return Number(amount);
+  if (unit === '') return Math.ceil(Number(amount));
```

Comment:
I don't agree with you.
`1.0000000000000001` is not a valid number of bytes. It's a number of bytes that is greater than `1` and lower than `2`. That means it's either `1` or `2`, we have no way to know. But in the theoretical extreme case we have only 1 byte left on our hard drive, I think taking the risk to say "This file is only 1 byte long". If in reality it's 2 bytes long, we will be in big trouble.

NB: Of course, it's only a theoretical example, we cannot write a file of 1 byte with 1 byte left on the disk, because of metadata.